src/oidc/discovery.ts

```diff
@@ -38,23 +39,27 @@ export const discoverAndValidateOIDCIssuerWellKnown = async (issuer: string): Pr
         signal: timeoutSignal(5000),
     });
     const issuerWellKnown = await issuerWellKnownResponse.json();
-    const validatedIssuerConfig = validateOIDCIssuerWellKnown(issuerWellKnown);
+    return validateAuthMetadataAndKeys(issuerWellKnown);
+};
+
+/**
+ *
```

Comment:
Could do with some more doc

Reply:
Oops, forgot to fill, thanks for the spot